Documentation/ignition.md

```diff
@@ -1,16 +1,16 @@
-# Ignition
```

Comment:
Its moved from ignition.md to container-linux-config.md in the last commit. I figured it was easier to review if the change and move were separate.